test/over_react/util/map_util_test.dart

```diff
@@ -253,5 +254,54 @@ main() {
         expect(newStyleFromProps(domProps()), equals({}));
       });
     });
+
+    group('getBackingMap()', () {
+      group('when passed in a props object', () {
+        JsBackedMap testMap = JsBackedMap.from({'id': 'test'});
+
+        var props = getBackingMap(TestProps(testMap));
+
+        test('returns a map', () {
+          expect(props, TypeMatcher<Map>());
+        });
+
+        test('returns the correct custom map values', () {
+          expect(props.containsKey('id'), isTrue);
+          expect(props['id'], 'test');
+        });
+      });
+
+      group('when passed in a state object', () {
+        JsBackedMap testMap = JsBackedMap.from({'isActive': true});
+
+        var state = getBackingMap(TestState(testMap));
+
+        test('returns a map', () {
+          expect(state, TypeMatcher<Map>());
+        });
+
+        test('returns the correct custom map values', () {
+          expect(state.containsKey('isActive'), isTrue);
+          expect(state['isActive'], true);
+        });
+      });
```

Comment:
Gah, sorry, I didn't see this the first time around... but could we add test cases for `JsBackedMap` and `Map` as well, to provide full coverage?

Reply:
No worries! Of course!